src/foam/nanos/alarming/AlarmLoggingDAO.js

```diff
@@ -48,6 +48,7 @@ foam.CLASS({
           if ( alarm.getIsActive() ) {
             Notification notification = new Notification.Builder(x)
               .setTemplate("NOC")
+              .setEmailName("NOC")
```

Comment:
This is on the template.  It's not needed here. that's the whole point of the notification template.